client/dashboard/sites/backups/fields.tsx

```diff
@@ -0,0 +1,51 @@
+import { Icon } from '@wordpress/components';
+import { dateI18n } from '@wordpress/date';
+import { __ } from '@wordpress/i18n';
+import { cloud, globe, image } from '@wordpress/icons';
+import type { ActivityLogEntry } from '../../data/types';
+import type { Field } from '@wordpress/dataviews';
+
+function getIcon( item: ActivityLogEntry ) {
```

Comment:
Nit: I wonder if it's essentially the same logic as the entries in the existing Last Activity Card?

https://github.com/Automattic/wp-calypso/blob/946e47bf343fcfb92799f2585f7e8eee726b8649/client/dashboard/sites/overview-latest-activity-card/index.tsx#L20-L26